app/models/podcast_import.rb

```diff
@@ -148,6 +149,26 @@ def create_podcast
     podcast
   end
 
+  def enclosure_prefix(podcast_item)
+    prefix = ''
+    link = [podcast_item.feedburner_orig_enclosure_link,
```

Comment:
these matches work, but there is this regex geek in me that feels the need to say you can do both the match to see if there is blubrry, in there and get out the prefix info in one regex:
```
> l = "http://www.podtrac.com/pts/redirect.mp3/media.blubrry.com/99percentinvisible/dovetail.prxu.org/99pi/9350e921-b910-4b1c-bbc6-2912d79d014f/248-Atom-in-the-Garden-of-Eden.mp3"

# if there is a match, the blubrry prefix will be in match data `1`
> bp = l.match(/\/(media\.blubrry\.com\/[^\/]+)\//)
=> #<MatchData "/media.blubrry.com/99percentinvisible/" 1:"media.blubrry.com/99percentinvisible">
```

likewise for podtrac, you can get the podtrac specific subdomain and path:
```
> pp = l.match(/\/(\w+\.podtrac.com\/.+?\.mp3)/)
=> #<MatchData "/www.podtrac.com/pts/redirect.mp3" 1:"www.podtrac.com/pts/redirect.mp3">
```